### DISCUSSION TITLE
Broken Link in Documentation

### DISCUSSION THREAD

**Discussion by ph818** [pivpn/pivpn#1543] [Bug Reports]
I am trying to change the pivpnHOST from public IP to dyndns. But the link in the [FAQ](https://docs.pivpn.io/faq/) does not go anywhere. It is broken: https://github.com/pivpn/pivpn#how-do-i-change-the-public-ipdns-name-of-the-pivpn-after-the-install

Where can I get this information?

Also, I specifically chose "Use DNS name" instead of "Use Public IP" in the installer, so either I misclicked, or the installer used the external IP anyway.

Thank you for this software though! It is making it much quicker to set up the VPN.

**Comment by coolapso** [comment]
yup you are right ... I am a little short on time today ... But i will try to update those ASAP ... 

Here's the Link for wireguard: https://docs.pivpn.io/wireguard/#changing-the-public-ipdns 
here's the link for OpenVPN: https://docs.pivpn.io/openvpn/#changing-the-public-ipdns

**Reply by coolapso** [reply]
Opened Issue on docs repository: https://github.com/pivpn/docs/issues/12  

Should get it fixed either today or tomorrow

**Reply by coolapso** [reply]
Fixed: https://github.com/pivpn/docs/releases/tag/v1.2.2
